tests/test_utils_datatypes.py

```diff
@@ -242,6 +242,31 @@ def test_set(self):
         for v in [-3, "test", 1.1]:
             self.assertNotIn(v, d)
 
+
+class LocalCacheTest(unittest.TestCase):
+
+    def test_cache_with_limit(self):
+        cache = LocalCache(limit=2)
+        cache['a'] = 1
+        cache['b'] = 2
+        cache['c'] = 3
+        self.assertEqual(len(cache), 2)
+        self.assertNotIn('a', cache)
+        self.assertIn('b', cache)
+        self.assertIn('c', cache)
+        self.assertEqual(cache['b'], 2)
+        self.assertEqual(cache['c'], 3)
+
+    def test_cache_without_limit(self):
+        max = 10**4
```

Comment:
Why so high a number? Wouldn’t `2` or even `1` work?

Reply:
Sure, but it doesn't really make a difference:
```
$ time python tests/test_utils_datatypes.py  # 10**1
.......................
----------------------------------------------------------------------
Ran 23 tests in 0.002s

OK

real	0m0,314s
user	0m0,298s
sys	0m0,016s
```
```
$ time python tests/test_utils_datatypes.py  # 10**2
.......................
----------------------------------------------------------------------
Ran 23 tests in 0.002s

OK

real	0m0,323s
user	0m0,291s
sys	0m0,032s
```
```
$ time python tests/test_utils_datatypes.py  # 10**3
.......................
----------------------------------------------------------------------
Ran 23 tests in 0.004s

OK

real	0m0,303s
user	0m0,260s
sys	0m0,044s
```
```
$ time python tests/test_utils_datatypes.py  # 10**4
.......................
----------------------------------------------------------------------
Ran 23 tests in 0.029s

OK

real	0m0,353s
user	0m0,303s
sys	0m0,050s
```